api/api.go

```diff
@@ -120,12 +125,36 @@ func (h *httpHandler) initRouter() {
 	}
 }
 
-func makeHandler(ctrl libnetwork.NetworkController, fct processor) http.HandlerFunc {
+const (
+	// only one Accept value for now
+	acceptRegex  = `^application/vnd\.libnetwork\.v\d\.\d\+json$`
```

Comment:
Should this be, e.g. `application/vnd.docker.libnetwork.....`?

A quick search on this subject;
http://www.w3.org/Protocols/rfc2616/rfc2616-sec3.html#sec3.7

> Media-type values are registered with the Internet Assigned Number Authority (IANA [19]). 
> The media type registration process is outlined in RFC 1590 [http://www.w3.org/Protocols/rfc2616/rfc2616-sec17.html#bib17]. *Use of non-registered media types is discouraged*